### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jlaura**
Updates README to remove the ISIS3 header and to include the Zenodo DOI for citing our work.

## Description

Update to add citation link.

## Related Issue

#3810 
## Motivation and Context

This change improves the ability of users to cite our work in academic publications.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-# ISIS3
+# ISIS
```
Comment (jessemapel) [inline README.md]: We should go through and scrub the entire Readme of ISIS3 references at once. I'd rather not do this piece-meal.

**Comment by thareUSGS** [comment]
In Zenodo title has "Releas". Author list is a little odd, e.g. isis3mgr. Perhaps list in alpha order or should we just use Astrogeology Team (or Science Center) and link to contributor page...

Currently the recommended cite is (very odd):
Tracie Sucharski, Jesse Mapel, jcwbacker, Kristin, Kaitlyn Lee, AgoinsUSGS, … Ross Beyer. (2020, March 4). USGS-Astrogeology/ISIS3: ISIS3.10.1 Public Releas (Version 3.10.1). Zenodo. http://doi.org/10.5281/zenodo.3697255

Can we also acknowledge NASA or funding opp.

**Comment by jessemapel** [comment]
@thareUSGS I've considered putting release notes into the repo anyway. If we want them on both the astrodicuss post and zenodo that would be a good way to handle that.

**Comment by jessemapel** [comment]
@thareUSGS This PR is adding a link to our current Zenodo, do you want to move your comments over to the issue about citing ISIS #3810? We're trying to keep PR discussion to just about the changes in the PR.

File: README.md (hunk 1 above)
Comment (jlaura) [reply README.md]: I going to revert this and we can put in a different PR to do that.

**Comment by jlaura** [comment]
@jessemapel This is good for re-review. I removed the ISIS3 change because I don't want to tackle eh whole doc in a PR about Zenodo.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**